.gitignore

```diff
@@ -1,3 +1,10 @@
 /build
 /node_modules
 /data
+cordova/*/
```

Comment:
Pune te rog `/` la începutul path-urilor, ca să facă match doar în rădăcina proiectului, și scoate liniile goale de la sfârșit.